aten/src/ATen/CPUGeneratorImpl.cpp

```diff
@@ -127,8 +127,8 @@ void CPUGeneratorImpl::set_state(const c10::TensorImpl& new_state) {
   using detail::CPUGeneratorImplState;
   using detail::CPUGeneratorImplStateLegacy;
 
-  static_assert(std::is_pod<CPUGeneratorImplStateLegacy>::value, "CPUGeneratorImplStateLegacy is not a PODType");
-  static_assert(std::is_pod<CPUGeneratorImplState>::value, "CPUGeneratorImplState is not a PODType");
+  static_assert(c10::is_pod_v<CPUGeneratorImplStateLegacy>, "CPUGeneratorImplStateLegacy is not a PODType");
+  static_assert(c10::is_pod_v<CPUGeneratorImplState>, "CPUGeneratorImplState is not a PODType");
```

Comment:
Standard layout only (we are copying in and out this struct from C code; whether or not initialization is trivial doesn't really matter)